cmd/manager/main.go

```diff
@@ -63,7 +67,13 @@ func main() {
 	features.OperatorControllerFeatureGate.AddFlag(pflag.CommandLine)
 	pflag.Parse()
 
+	if operatorControllerVersion {
+		fmt.Println(version.String())
+		os.Exit(0)
+	}
+
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts), zap.StacktraceLevel(zapcore.DPanicLevel)))
+	setupLog.Info("starting up the controller", "Git commit", version.String())
```

Comment:
Nit: This isn't just the git commit so that might be a bit misleading in the logs. Maybe something like:
```suggestion
	setupLog.Info("starting up the controller", "version info", version.String())
```

Reply:
Good catch. fixed it.